lale/operators.py

```diff
@@ -452,6 +453,63 @@ def pretty_print(
             markdown = IPython.display.Markdown(f"```python\n{result}\n```")
             return IPython.display.display(markdown)
 
+    def diff(
+        self,
+        other: Union[Any, "Operator"],
+        show_imports: bool = True,
+        customize_schema: bool = True,
+        ipython_display: bool = True,
```

Comment:
I would default `ipython_display` to False just like in `pretty_print`.

Reply:
By the same token, I'm assuming `customize_schema` should default to False too then.